app/boards/shields/elephant42/elephant42.zmk.yml

```diff
@@ -0,0 +1,14 @@
+file_format: "1"
```

Comment:
@filoxo Please run prettier on this file! See here: https://github.com/zmkfirmware/zmk/runs/4998524969?check_suite_focus=true

Reply:
I finally got around to fixing the formatting. Thanks!